plugins/database/postgresql/postgresql.go

```diff
@@ -26,6 +26,10 @@ ALTER ROLE "{{name}}" VALID UNTIL '{{expiration}}';
 `
 	defaultPostgresRotateRootCredentialsSQL = `
 ALTER ROLE "{{username}}" WITH PASSWORD '{{password}}';
+`
+
+	defaultPostgresRotateCredentialsSQL = `
+ALTER ROLE "{{name}}" WITH PASSWORD '{{password}}';
```

Comment:
Should this be `username` like the one above?

Reply:
The SQL statement for rotating root credentials uses `username`, but the statements for Roles use `name`:

- https://www.vaultproject.io/api/secret/databases/postgresql.html

- create user code: https://github.com/hashicorp/vault/blob/9082d08cd09b9a3bcc41b530ac79ef687d9204ef/plugins/database/postgresql/postgresql.go#L143
- rotate root code: https://github.com/hashicorp/vault/blob/9082d08cd09b9a3bcc41b530ac79ef687d9204ef/plugins/database/postgresql/postgresql.go#L406

I agree that it's confusing with both, but using `name` here is consistent with dynamic Roles